apps/src/templates/instructions/ScrollButtons.jsx

```diff
@@ -52,25 +52,36 @@ class ScrollButtons extends React.Component {
     return scrollButtonsHeight + this.getMargin() * 2;
   }
 
-  scrollStartUp = () => {
-    this.scrollStart(DIRECTIONS.UP);
+  continuousScrollStartUp = () => {
+    this.continuousScrollStart(DIRECTIONS.UP);
   };
 
-  scrollStartDown = () => {
-    this.scrollStart(DIRECTIONS.DOWN);
+  continuousScrollStartDown = () => {
+    this.continuousScrollStart(DIRECTIONS.DOWN);
   };
 
-  scrollStart(dir) {
+  singleScrollUp = () => {
+    this.singleScroll(DIRECTIONS.UP);
+  };
+
+  singleScrollDown = () => {
+    this.singleScroll(DIRECTIONS.DOWN);
+  };
+
+  singleScroll(dir) {
```

Comment:
Interesting observation that when a user clicks and holds the up/down button, `singleScroll` is repeatedly called on acting like a continuous scroll.